misk-crypto/src/test/kotlin/misk/crypto/CryptoModuleTest.kt

```diff
@@ -123,7 +125,10 @@ class CryptoModuleTest {
       }
     )
     val config = CryptoConfig(listOf(key), "test_master_key")
-    val injector = Guice.createInjector(CryptoTestModule(config), DeploymentModule.forTesting())
+    val env = Env(TESTING.name)
+    val deploymentModule = DeploymentModule(TESTING, env)
```

Comment:
could be a convenience function:

`DeploymentModule.test() = DeploymentModule(TESTING, Env(TESTING.name))`

I see some of the tests use `Env` directly, how about `TESTING.env()`?

Reply:
I'll keep it in mind.  I'm going to try to remove Env as well first.

Reply:
Env has been removed.